src/routes/docs/products/databases/queries/+page.markdoc

```diff
@@ -418,3 +418,294 @@ query {
+            Query.lessThan("age", value: 65),
+            Query.equal("isActive", value: true)
+        ])
+    ]
+)
+```
+```kotlin
+val results = databases.listDocuments(
+    databaseId = '<DATABASE_ID>',
+    collectionId = '<COLLECTION_ID>',
+    queries = listOf(
+        Query.and(listOf(
+            Query.greaterThan("age", 18),
+            Query.lessThan("age", 65),
+            Query.equal("isActive", true)
+        ))
+    )
+)
+```
+{% /multicode %}
+
+The `Query.and()` method takes an array of queries and requires all conditions to be true. This example will return documents where the age is between 18 and 65 (exclusive) AND the user is active.
+
+## OR Queries
+Use `Query.or()` when you want to match documents that satisfy any of the specified conditions. For example, to find items in either the books or magazines category:
+
+{% multicode %}
+```client-web
+const results = await databases.listDocuments(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    [
+        Query.or([
+            Query.equal('category', 'books'),
+            Query.equal('category', 'magazines')
+        ])
+    ]
+);
+```
+```client-flutter
+final results = await databases.listDocuments(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    [
+        Query.or([
+            Query.equal('category', 'books'),
+            Query.equal('category', 'magazines')
+        ])
+    ]
+);
+```
+```python
+results = databases.list_documents(
+    database_id='<DATABASE_ID>',
+    collection_id='<COLLECTION_ID>',
+    queries=[
+        Query.or_queries([
+            Query.equal('category', 'books'),
+            Query.equal('category', 'magazines')
+        ])
+    ]
+)
+```
+```client-apple
+let results = try await databases.listDocuments(
+    databaseId: '<DATABASE_ID>',
+    collectionId: '<COLLECTION_ID>',
+    queries: [
+        Query.or([
+            Query.equal("category", value: "books"),
+            Query.equal("category", value: "magazines")
+        ])
+    ]
+)
+```
+```kotlin
+val results = databases.listDocuments(
+    databaseId = '<DATABASE_ID>',
+    collectionId = '<COLLECTION_ID>',
+    queries = listOf(
+        Query.or(listOf(
+            Query.equal("category", "books"),
+            Query.equal("category", "magazines")
+        ))
+    )
+)
+```
+{% /multicode %}
+
+The `Query.or()` method takes an array of queries and matches documents that satisfy any of the conditions. This example will return documents where the category is either "books" OR "magazines".
+
+## Complex Queries
+You can create complex queries by combining AND and OR operations. For example, to find items in either the books or magazines category that are priced between $10 and $100:
+
+{% multicode %}
+```client-web
+const results = await databases.listDocuments(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    [
+        Query.or([
+            Query.equal('category', 'books'),
+            Query.equal('category', 'magazines')
+        ]),
```

Comment:
You'd typically do `Query.equal('category', ['books', 'magazines'])` instead so can we have a more realistic example?

Reply:
Updated